mlir/include/mlir/Dialect/MIOpen/MIOpenOps.td

```diff
@@ -159,7 +158,6 @@ def MIOpen_GridwiseGemmV2Op :
                    BoolArrayAttr:$aOobDims,
                    BoolArrayAttr:$bOobDims,
                    BoolArrayAttr:$cOobDims,
-                   ArgTransforms<3>:$transforms,
                    MIOpen_PaddingInfoAttr:$paddingInfo,
                    InMemoryDataOperation:$storeOperation)> {
```

Comment:
Can we consolidate `storeOperation` used in `gridwise_gemm_v2`, and `dataOperation` in `buffer_store` into the same name?

Reply:
Got any thoughts on which of the names makes more sense? Or if we should go all-in on the renamings and make it all something like `storeMethod`?

On the other hand, the advantage of the current scheme is that on gridwise_gemm_v2 it's clear the enum is about how the data will be stored, while in `buffer_store` (where we know it's a store) we don't do anything redundant. Maybe `storeOperation` on gemm and `method` on the actual store operation?

Reply:
The enum itself might deserve a better name. Perhaps `InMemoryDataOperation` -> `DataStoreMethod`. Then both `storeOperand` and `dataOperand` can simply be called `method`.

Reply:
I went with just `StoreMethod`